test/specs/scenarios/document.js

```diff
@@ -56,7 +56,7 @@ export const documentScenarios = async (lang) => {
         passportUploadImageGuide.upload(filename)
       }
 
-      it('should display document upload screen on desktop browsers when useLiveDocumentCapture is enabled', async () => {
+      it('should display document upload screen on desktop browsers when useLiveDocumentCapture is enabled @percy', async () => {
```

Comment:
Just wondering, does the tag have to be in this format? Thinking `+percy` might be a better tag. But not a big deal.

Reply:
Testing best practices use the '@' annotation, its generally considered a standardised way of tagging tests. That's the reason why its used in this instance.